app.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const {
+  getObjects, getarticles,
+  } = require("./controllers/news.controller");
+const app = express();
+app.use(express.json());
```

Comment:
_Really_ you should only add middleware etc. as and when required but I understand app.use(express.json());  is easy to forget.